src/main/kotlin/org/jitsi/nlj/util/ArrayCache.kt

```diff
@@ -63,16 +63,23 @@ open class ArrayCache<T>(
     /**
      * Inserts an item with a specific index in the cache. Stores a copy.
      */
-    fun insertItem(item: T, index: Int): Boolean =
+    fun insertItem(item: T, index: Int, timeAdded: Long): Boolean =
```

Comment:
Should we use an `Instant` for the timestamp while we're here?  Or is that just overcomplicating things?

Reply:
I feel like it's overcomplicating things, especially since this approach has two copies of the time, and `Instant` is an object.